app/src/main/java/com/sjh/autosummary/feature/AutoSummaryApp.kt

```diff
@@ -34,7 +34,7 @@ fun AutoSummaryNavHost(
     ) {
         mainScreen(onHistoryClick = navController::navigateToHistory, dataKey = HISTORYDATA_KEY)
         historyScreen(
-            onChatHistoryClick = { messageList -> navController.navigateToMain(messageList) },
+            onChatHistoryClick = { chatHistoryId -> navController.navigateToMain(chatHistoryId) },
```

Comment:
```suggestion
            onChatHistoryClick = navController::navigateToMain,
```
이렇게 혹시 안될까요?
위의 mainScreen에서는 사용하고 있는 것 같은데요.

Reply:
제가 이 부분과 관련된 코틀린 문법을 잘 몰랐습니다. 수정해두고 관련 예시 코드들을 확인해보겠습니다!